client/src/components/viewers/FileControlsButton.vue

```diff
@@ -0,0 +1,64 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-button
+    class="file-controls-button button-medium"
+    size="default"
+  >
+    <ion-icon
+      class="file-controls-button-icon"
+      v-if="icon"
+      slot="start"
+      :icon="icon"
+    />
+    <ms-image
+      class="file-controls-button-icon"
+      v-if="!icon && image"
+      :image="image"
+    />
+    {{ $msTranslate(buttonLabel) }}
+  </ion-button>
+</template>
+
+<script setup lang="ts">
+import { MsImage, Translatable } from 'megashark-lib';
+import { IonButton, IonIcon } from '@ionic/vue';
+
+defineProps<{
+  buttonLabel?: Translatable;
+  icon?: string;
+  image?: string;
+  isDropdown?: boolean;
```

Comment:
I'm guessing the dropdown is here to add an arrow. Be careful with that, you'll probably want to invert the arrow when the dropdown is opened and you'll be entering a world of pain. Probably better to do a separated dropdown-button if that's the case.